### PR TITLE
Add actual terms! Woo!

### LINKED ISSUE #219: Specify text to be used on Terms page
As any EncoreLink user, I need to know what are the terms of use for the site.

### PR DESCRIPTION
**Pull request by wdoug**
This PR closes #219 

#### What does this PR do?

Adds actual terms to the terms modal! Thanks, @JonathanLayman!

### CODE REVIEW

File: client/src/components/Terms.js [hunk 1]
```diff
@@ -3,8 +3,125 @@ import React from 'react';
...
+          These Website Standard Terms and Conditions written on this webpage shall manage your use
+          of this website. These Terms will be applied fully and affect to your use of this Website.
+          By using this Website, you agreed to accept all terms and conditions written in here. You
+          must not use this Website if you disagree with any of these Website Standard Terms and
+          Conditions.
+        </p>
+        <p>
+          Minors or people below 18 years old are not allowed to use this Website.
+        </p>
+        <ol start={2}>
```
Comment (dviramontes) [inline client/src/components/Terms.js:19]: neat!

**Review by dviramontes — APPROVED** [review]
👍 lgtm!

File: client/src/components/Terms.js [hunk 2]
```diff
@@ -3,8 +3,125 @@ import React from 'react';
...
+          <li><strong>Intellectual Property Rights</strong></li>
+        </ol>
+        <p>
+          Other than the content you own, under these Terms, EncoreLInk and/or its licensors own all
```
Comment (nlkluth) [inline client/src/components/Terms.js]: is the capital "I" intentional?

Comment (dviramontes) [reply client/src/components/Terms.js]: Good catch!

**Comment by JonathanLayman** [comment]
Terms reviewed and approved. Thanks for your help everyone!

**Pull request merged**